pytest_fixtures/xdist.py

```diff
@@ -0,0 +1,12 @@
+"""Fixtures specific to or relating to pytest's xdist plugin"""
+import pytest
+
+from robottelo.config import settings
+
+
+@pytest.fixture(scope="session", autouse=True)
+def align_xdist_satellites(worker_id):
+    """Set a different Satellite per worker when available in robottelo's config"""
+    settings.server.hostname = settings.server.get_hostname(worker_id)
+    settings._configure_entities()
```

Comment:
The `settings.configure()` above will automatically take care of `_configure_entities()` and `_configure_airgun()`. So we are good to remove it from here.